From 1afe7e4fcc6c699d161f6e077b70d57844ab0602 Mon Sep 17 00:00:00 2001
From: Rami Potinkara <rami.potinkara@qt.io>
Date: Wed, 3 Sep 2025 10:22:41 +0300
Subject: [PATCH 200/239] Android: update Qt Quick for Android examples to SDK
 36

This patch updates qtdeclarative examples to use
SDK 36 as a target and compile version SDK.

Task-number: QTBUG-138552
Task-number: QTBUG-138549
Change-Id: If655c6654790b4be3704325aceca6df4947362d4
Reviewed-by: Nicholas Bennett <nicholas.bennett@qt.io>
Reviewed-by: Rami Potinkara <rami.potinkara@qt.io>
---
 .../android/models/qtabstractitemmodel_java/app/build.gradle  | 4 ++--
 .../models/qtabstractlistmodel_kotlin/app/build.gradle.kts    | 4 ++--
 examples/platforms/android/qtquickview_java/app/build.gradle  | 4 ++--
 .../platforms/android/qtquickview_kotlin/app/build.gradle.kts | 4 ++--
 src/quick/platform/android/jar/build.gradle                   | 2 +-
 .../platforms/android/qtquickview_java/app/build.gradle       | 4 ++--
 6 files changed, 11 insertions(+), 11 deletions(-)

diff --git a/examples/platforms/android/models/qtabstractitemmodel_java/app/build.gradle b/examples/platforms/android/models/qtabstractitemmodel_java/app/build.gradle
index 40b201b849..05d9664eb8 100644
--- a/examples/platforms/android/models/qtabstractitemmodel_java/app/build.gradle
+++ b/examples/platforms/android/models/qtabstractitemmodel_java/app/build.gradle
@@ -11,12 +11,12 @@ QtBuild {
 
 android {
     namespace 'com.example.qtabstractitemmodel_java'
-    compileSdk 35
+    compileSdk 36
 
     defaultConfig {
         applicationId "com.example.qtabstractitemmodel_java"
         minSdk 28
-        targetSdk 35
+        targetSdk 36
         versionCode 1
         versionName "1.0"
 
diff --git a/examples/platforms/android/models/qtabstractlistmodel_kotlin/app/build.gradle.kts b/examples/platforms/android/models/qtabstractlistmodel_kotlin/app/build.gradle.kts
index bfeb8e117d..dae916f50e 100644
--- a/examples/platforms/android/models/qtabstractlistmodel_kotlin/app/build.gradle.kts
+++ b/examples/platforms/android/models/qtabstractlistmodel_kotlin/app/build.gradle.kts
@@ -14,12 +14,12 @@ QtBuild {
 
 android {
     namespace = "com.example.qtabstractlistmodel_kotlin"
-    compileSdk = 35
+    compileSdk = 36
 
     defaultConfig {
         applicationId = "com.example.qtabstractlistmodel_kotlin"
         minSdk = 28
-        targetSdk = 35
+        targetSdk = 36
         versionCode = 1
         versionName = "1.0"
 
diff --git a/examples/platforms/android/qtquickview_java/app/build.gradle b/examples/platforms/android/qtquickview_java/app/build.gradle
index 83ab07cc84..c53764c773 100644
--- a/examples/platforms/android/qtquickview_java/app/build.gradle
+++ b/examples/platforms/android/qtquickview_java/app/build.gradle
@@ -11,12 +11,12 @@ QtBuild {
 
 android {
     namespace 'com.example.qtquickview_java'
-    compileSdk 35
+    compileSdk 36
 
     defaultConfig {
         applicationId "com.example.qtquickview_java"
         minSdk 28
-        targetSdk 35
+        targetSdk 36
         versionCode 1
         versionName "1.0"
 
diff --git a/examples/platforms/android/qtquickview_kotlin/app/build.gradle.kts b/examples/platforms/android/qtquickview_kotlin/app/build.gradle.kts
index c7f92d5704..b7103c4c4c 100644
--- a/examples/platforms/android/qtquickview_kotlin/app/build.gradle.kts
+++ b/examples/platforms/android/qtquickview_kotlin/app/build.gradle.kts
@@ -12,12 +12,12 @@ QtBuild {
 
 android {
     namespace = "com.example.qtquickview_kotlin"
-    compileSdk = 35
+    compileSdk = 36
 
     defaultConfig {
         applicationId = "com.example.qtquickview_kotlin"
         minSdk = 28
-        targetSdk = 35
+        targetSdk = 36
         versionCode = 1
         versionName = "1.0"
 
diff --git a/src/quick/platform/android/jar/build.gradle b/src/quick/platform/android/jar/build.gradle
index ea6d06c257..9e152d13c3 100644
--- a/src/quick/platform/android/jar/build.gradle
+++ b/src/quick/platform/android/jar/build.gradle
@@ -23,7 +23,7 @@ repositories {
 }
 
 android {
-    compileSdk 34
+    compileSdk 36
 
     defaultConfig {
         minSdkVersion 28
diff --git a/tests/manual/platforms/android/qtquickview_java/app/build.gradle b/tests/manual/platforms/android/qtquickview_java/app/build.gradle
index dca4095dac..f82a71ec2d 100644
--- a/tests/manual/platforms/android/qtquickview_java/app/build.gradle
+++ b/tests/manual/platforms/android/qtquickview_java/app/build.gradle
@@ -26,12 +26,12 @@ android {
         }
     }
     namespace 'com.example.qtquickview_java'
-    compileSdk 35
+    compileSdk 36
 
     defaultConfig {
         applicationId "com.example.qtquickview_java"
         minSdk 28
-        targetSdk 35
+        targetSdk 36
         versionCode 1
         versionName "1.0"
 
-- 
2.51.2

